portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -20,13 +20,31 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import com.google.gson.Gson;
+import java.util.ArrayList;
+
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    // response.setContentType("text/html;");
+    //response.getWriter().println("Hello Shayne!");
+
+    ArrayList<String> myArr = new ArrayList<String>();
+    myArr.add("First");
+    myArr.add("Second");
+    myArr.add("Third");
+
+    String json = convertToJson(myArr);
```

Comment:
Perhaps a more meaningful name would help the readers understand the purpose of the variable.